.gitmodules

```diff
@@ -1,12 +1,6 @@
 [submodule "kucher/libraries/popcop"]
 	path = kucher/libraries/popcop
 	url = https://github.com/Zubax/popcop
-[submodule "kucher/libraries/construct"]
-	path = kucher/libraries/construct
-	url = https://github.com/construct/construct
 [submodule "kucher/libraries/dataclasses"]
 	path = kucher/libraries/dataclasses
-	url = https://github.com/ericvsmith/dataclasses
-[submodule "kucher/libraries/quamash"]
-	path = kucher/libraries/quamash
-	url = https://github.com/harvimt/quamash
+	url = https://github.com/ericvsmith/dataclasses
```

Comment:
This should no longer be needed because in Python 3.7 onwards this is part of the standard library.

Reply:
It is an accident, but I want to fork them and have our own copies

Reply:
You can do it but what for? These are relatively popular libraries and there should be little risk of them disappearing overnight (assuming you are talking about construct and quamash). Even if we did fork them, surely we don't want to bother with maintaining independent packages on PyPI.